items/src/test/scala/weco/api/items/fixtures/ItemsApiFixture.scala

```diff
@@ -28,28 +30,48 @@ trait ItemsApiFixture extends SierraSourceFixture {
       publicRootPath = "/catalogue"
     )
 
+  def withClock[R](
+    dateTime: String = "2024-04-24T11:00:00.000Z"
+  )(testWith: TestWith[Clock, R]): R = {
+    val instant = Instant.parse(dateTime)
+    val zoneId = ZoneId.of("Europe/London")
+    testWith(Clock.fixed(instant, zoneId))
+  }
+
   def withItemsApi[R](
     catalogueResponses: Seq[(HttpRequest, HttpResponse)] = Seq(),
-    sierraResponses: Seq[(HttpRequest, HttpResponse)] = Seq()
+    sierraResponses: Seq[(HttpRequest, HttpResponse)] = Seq(),
+    contentApiVenueResponses: Seq[(HttpRequest, HttpResponse)] = Seq()
   )(testWith: TestWith[Unit, R]): R =
     withActorSystem { implicit actorSystem =>
-      withSierraSource(sierraResponses) { sierraSource =>
-        val itemsUpdaters = List(
-          new SierraItemUpdater(sierraSource)
-        )
-
-        val catalogueApiClient = new MemoryHttpClient(catalogueResponses)
-        with HttpGet {
-          override val baseUri: Uri = Uri("http://catalogue:9001")
-        }
+      withClock() { clock =>
```

Comment:
We don't really care about the availableDates at that level so using the default in the fixture, instead of wrapping all the tests with `withClock`